integration/integration.go

```diff
@@ -49,6 +49,19 @@ type IdentifierAttribute struct {
 	value string
 }
 
+// NewIDAttr creates new identifier attribute.
+func NewIDAttr(key, value string) IdentifierAttribute {
```

Comment:
Since this `new` method is only used in tests and does not avoid a lot of complexity I would go for creating `map[string][string]`. Less is more ;-P